mono/utils/mach-support-ppc64.c

```diff
@@ -0,0 +1,139 @@
+/**
+ * \file
+ * mach support for PPC
+ *
+ * Authors:
+ *   Sergey Fedorov (vital.had@gmail.com)
+ *
+ * (C) 2023 Xamarin, Inc.
+ */
+
+/* The code below assumes 10.5.x.
```

Comment:
could be dropped; IIRC ppc64 was introduced in 10.5

Reply:
AFAIK 10.4.11 does support `ppc64`, at least to some extent.